.gitlab-ci.yml

```diff
@@ -280,32 +283,32 @@ publish-polkadot-image:
     - job:                         test-build-linux-stable
       artifacts:                   true
 
-publish-adder-collator-image:
-  # service image for Simnet
-  stage:                           build
-  <<:                              *build-push-image
-  variables:
-    <<:                            *image-variables
-    # scripts/docker/collator.Dockerfile
-    DOCKERFILE:                    collator.Dockerfile
-    IMAGE_NAME:                    docker.io/paritypr/colander
-  rules:
-    - if: $CI_PIPELINE_SOURCE == "schedule"
-    - if: $CI_COMMIT_REF_NAME == "master"
-    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
-    - if: $CI_COMMIT_REF_NAME == "rococo-v1"
-  needs:
-    - job:                         build-adder-collator
-      artifacts:                   true
-  after_script:
-    - buildah logout "$IMAGE_NAME"
-    # pass artifacts to the trigger-simnet job
-    - echo "COLLATOR_IMAGE=$IMAGE_NAME" > ./artifacts/collator.env
-    - echo "COLLATOR_IMAGE_TAG=$(cat ./artifacts/EXTRATAG)" >> ./artifacts/collator.env
-  artifacts:
-    reports:
-      # this artifact is used in trigger-simnet job
-      dotenv: ./artifacts/collator.env
+# publish-adder-collator-image:
```

Comment:
Nit: Perhaps we can leave a comment when we expect this to be uncommented?

Reply:
`publish-adder-collator-image` is fully commented as `trigger-simnet`, implicitly implying that these should be re-enabled entirely. The TODO comment for commented part of `publish-polkadot-image` job only points to the specific rules related to the simnet stuff.

If first two commented jobs should be noted with their re-enablement condition, I can submit another PR. This one was already merged.